src/etc/inc/system.inc

```diff
@@ -1059,6 +1059,22 @@ EOD;
 		$syslogd_extra = "-f {$g['varetc_path']}/syslog.conf {$sourceip}";
 	}
 
+	$log_sockets = array();
+	$log_sockets[] = "{$g['dhcpd_chroot_path']}/var/run/log";
+
+	if (isset($config['installedpackages']['package'])) {
+		foreach ($config['installedpackages']['package'] as $package) {
+			if (isset($package['logging']['logsocket']) && '' != $package['logging']['logsocket'] &&
```

Comment:
Could you please reverse the comparison from:
```
'' != $var
```
to
```
$var != ''
```
to match the style used in the whole project